src/MessageContainer.js

```diff
@@ -135,10 +135,20 @@ export default class MessageContainer extends React.Component {
     );
   }
 
+  endReachedProps(){
+    let condProps = {}
+    if (this.props.onTopReached){ condProps.onEndReached = this.props.onTopReached }
```

Comment:
@dgdavid I think this is the right way, the underlaying scroll component is an `InvertibleScrollView` so the top is the end.
@arturictus Am I getting this right?